modules/51_internalizeDamages/TCitr/declarations.gms.nash

```diff
@@ -0,0 +1,20 @@
+*** |  (C) 2006-2020 Potsdam Institute for Climate Impact Research (PIK)
```

Comment:
Is it an error that this filename ends with .gms.nash?`

Reply:
no, it was "stored" and I deleted it.